tests/unit/clients/cassandra_tests.py

```diff
@@ -86,3 +103,212 @@ def test_prepare(self):
         # prepares.
         calls = [mock.call(statement), mock.call(statement)]
         self.session.prepare.assert_has_calls(calls)
+
+    def test_execute_async_prom_metrics(self):
+        self.session.keyspace = "keyspace"  # mocking keyspace name
+        self.session.cluster.contact_points = [
+            "hostname1",
+            "hostname2",
+        ]  # mocking cluster contact points
+        self.adapter.execute_async("SELECT foo from bar;")
+
+        requests_active = REGISTRY._names_to_collectors[
```

Comment:
Could you [use `REGISTRY.get_sample_value` instead](https://sourcegraph.com/github.com/prometheus/client_python/-/blob/prometheus_client/registry.py?L131) here?

Reply:
Thanks I couldn't find this function and nobody raised it when we used this approach in other places.
Will update this PR and do a follow up update for the rest of the codebase.